lib/ingestors/ingestor_event_rest.rb

```diff
@@ -401,6 +417,226 @@ def process_elixir(url)
       end
     end
 
+    def process_vu(url)
+      # execute REST request
+      results = get_JSON_response url
+      data = results.to_h['results']
+
+      # extract materials from results
+      unless data.nil? or data.size < 1
+        data.each do |item|
+          begin
+            # create new event
+            event = Event.new
+
+            # extract event details from
+            attr = item
+            event.title = attr['title']
+            event.url = attr['url'].strip unless attr['url'].nil?
+            event.organizer = attr['org']
+            event.description = convert_description attr['description']
+            event.start = attr['startdt']
```

Comment:
what happens if one of the fields is not filled in? a keyError probably. Perhaps eventbrite ensures it's never the case though. It may be nicer to use fetch with a default argument.